src/components/LoginTitle/LoginTitle.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// import Theme from '../../styles/theme';
+
+function LoginTitle() {
+  return (
+    <TitleMargin margin="0px 0px 40px 0px">
+      <Title fontSize="50px" fontWeight="bold">
+        WESELL
+      </Title>
+      <SubTitle margin="10px 0px" fontSize="15px" fontWeight="bold">
+        We Ensure Society Effects L L
+      </SubTitle>
+    </TitleMargin>
+  );
+}
+
+const TitleMargin = styled.div`
```

Comment:
```suggestion
const Container = styled.div`
```
이 이름은 내용 + 구현 이런 구성이라 어색한거 같아요 그냥 Title이라고 쓰든가 저는 스타일드 컴포넌트 바깥 쪽에 있는건 그냥 Container라고 씁니다

Reply:
Wrapper나 Container로 사용하도록 하겠습니다